lib/frodo/version.rb

```diff
@@ -1,3 +1,3 @@
 module Frodo
-  VERSION = '0.10.8'
+  VERSION = '0.11.0'
```

Comment:
Bumping the minor version since the public method `referential_constraints` has been removed in favor of `referential_constraints_for_entity`-- I would be shocked if anyone notices.